third_party/move/move-compiler-v2/tests/reference-safety/v1-tests/mutate_full.exp

```diff
@@ -1,10 +0,0 @@
-
-Diagnostics:
-error: implicit copy of mutable reference in local `x` which is used later
```

Comment:
Being able to remove this error is nice, but I fear it may make it hard for users to reason about why it works.  Can you state a simple rule?  Will it still work if you had
```
     11         let x = &mut 0;
     12         let f = x; // error in v2 because of copy
     13         a = *f;
     14         *x = 0;
     15         b = *f;
```
Why not?  (Not for me, for a user.)

Reply:
Can you please check out the PR description? It contains the reasoning why it works. The above is still an error because even though you can copy mutref, while this copy is alive, the source cannot be used, as described in the PR header.  Here `x` is used before `f` goes out of scope.